plugins/regex_remap/regex_remap.cc

```diff
@@ -825,26 +838,32 @@ TSRemapNewInstance(int argc, char *argv[], void **ih, char * /* errbuf ATS_UNUSE
     }
 
     // Got a regex and substitution string
-    RemapRegex *cur = new RemapRegex();
+    std::unique_ptr<RemapRegex> cur(new RemapRegex);
 
     if (!cur->initialize(regex, subst, options)) {
       TSError("[%s] can't create a new regex remap rule", PLUGIN_NAME);
-      delete cur;
       continue;
     }
 
-    if (cur->compile(&error, &erroffset) < 0) {
-      TSError("[%s] PCRE failed in %s (line %d) at offset %d: %s", PLUGIN_NAME, (ri->filename).c_str(), lineno, erroffset, error);
-      delete (cur);
+    const char *error;
+    int erroffset;
+    if (cur->compile(error, erroffset) < 0) {
+      if (erroffset > 0)
```

Comment:
Also, here, we've moved to a standard of always using {}'s, so please add those even on these one liners.